be/src/udf/java/java_data_converter.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "column/binary_column.h"
+#include "column/column_visitor.h"
+#include "column/column_visitor_adapter.h"
+#include "column/fixed_length_column.h"
+#include "common/status.h"
+#include "common/statusor.h"
+#include "runtime/primitive_type.h"
+#include "udf/java/java_udf.h"
+
+namespace starrocks::vectorized {
+struct JavaUDAFState {
+    JavaUDAFState(jobject&& handle_) : handle(std::move(handle_)){};
+    ~JavaUDAFState() = default;
+    // UDAF State
+    jobject handle;
+};
+// Column to DirectByteBuffer, which could avoid some memory copy,
+// directly access the C++ address space in Java
+// Because DirectBuffer does not hold the referece of these memory,
+// we must ensure that it is valid during accesses to it
+
+class ConvertDirectBufferVistor : public ColumnVisitorAdapter<ConvertDirectBufferVistor> {
+public:
+    ConvertDirectBufferVistor(std::vector<DirectByteBuffer>& buffers) : ColumnVisitorAdapter(this), _buffers(buffers) {}
+    Status do_visit(const NullableColumn& column);
```

Comment:
find no implementation of this two method

Reply:
move from java_window_function.cpp to java_data_converter.cpp